app.rb

```diff
@@ -0,0 +1,157 @@
+require_relative 'student'
+require_relative 'teacher'
+require_relative 'book'
+require_relative 'rental'
+
+class App
+  def initialize
+    @books = []
+    @people = []
+    @rentals = []
+  end
+
+  def display_menu
+    puts "\n\nPlease choose an option by entering a number:"
+    puts '1 - List all books'
+    puts '2 - List all people'
+    puts '3 - Create a person'
+    puts '4 - Create a book'
+    puts '5 - Create a rental'
+    puts '6 - List all rentals for a given person id'
+    puts '7 - Exit'
+  end
```

Comment:
**strong points to resalt** ✔✨

Your code is very organized and clean, congrats on professionalism and how clear and clean is your code, keep doing a great work  

💪😄Happy coding!🌱

Reply:
## Hello @sheylaPozo ✋ 😃 
Thank you very much for your review and your comments 😃 . This helps me improve and move forward on my path as a software developer. 💪